Sources/MapboxMaps/Foundation/Extensions/Bundle+MapboxMaps.swift

```diff
@@ -11,7 +11,16 @@ extension Bundle {
         #if SWIFT_PACKAGE
         return Bundle.module
         #else
-        return Bundle(for: BundleLocator.self)
+        // When using frameworks this bundle will be our `.framework` bundle.
+        // When using static linking this bundle will be the the host application's `.app` bundle.
+        let bundle = Bundle(for: BundleLocator.self)
```

Comment:
Basically, this bundle will be used only for binary distribution, right?

I mean, for spm the module bundle will be used, for cocoapods, it will be the MapboxMapsResources.bundle, but for binary distribution we still need BundleLocator.

Reply:
Exactly!